src/plugins/maps_legacy/public/map/_legend.scss

```diff
@@ -33,3 +33,12 @@
   left: $euiSizeXXL;
   white-space: nowrap;
 }
+
+.mapButton {
+  display: inline-block;
+  font: normal normal normal 14px/1 FontAwesome, sans-serif;
```

Comment:
@AMoo-Miki - for a follow-up PR, is there a way we can reference a particular OUI icon directly via SASS/CSS for a case like this?

Reply:
This piece of code is not written the React way else they could have just used `OuiIcon`. There is no direct access to the imagery.

Reply:
Yeah, I figured. I'm not sure we'd even want to expose direct access to the assets as part of OUI.